### PR TITLE
Stop logging event data

### PR DESCRIPTION
**Pull request by rajdivya**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by joshuadeanhall — APPROVED** [review]

File: src/resource.ts [hunk 1]
```diff
@@ -586,7 +586,6 @@ export abstract class BaseResource<
                         this.prepareCredentialsFilter(this.callerSession)
                     );
                 }
-                this.log('EVENT DATA\n', eventData);
```
Comment (ammokhov) [inline src/resource.ts:589]: is that the only place?

**Review by ammokhov — APPROVED** [review]

File: src/resource.ts (hunk 1 above)
Comment (rajdivya) [reply src/resource.ts:589]: Yes, this is the only place logging event data in the repo.

Comment (rajdivya) [reply src/resource.ts:589]: Validated this is the only place logging event data by invoking CUD handlers of a resource.

**Review by ammokhov — APPROVED** [review]

**Review by joshuadeanhall — APPROVED** [review]

**Comment by agupta94** [comment]
Hi @rajdivya , what was the reasoning behind this? I use EVENT DATA log for various debugging reasons. It was helpful to get insight into how the cloudformation event looked like.

**Pull request merged**